lib/model/migrations/20220908-01-remove-google-backups.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = async (db) => {
+  await db.raw(`delete from sessions where "actorId" in (
+  select id from actors
+  where
+    type = 'singleUse' and
+    "displayName" = 'Backup creation token' and
```

Comment:
I'm not completely sure what the best way is to identify single-use backup creation tokens among actors, but it feels like this should be pretty robust. I also thought about checking that the `meta` column has a `keys` property and that the actor has an associated `initbkup` assignment. But ultimately I don't think those checks add much. I also don't think that there's a lot of risk here of unintentionally deleting other kinds of single-use actors (they'd have to be named "Backup creation token"!).